Paint Me

manognyaa opened the pull request:
## Related Issue 
I hope many of you people have painted a drawing in paper. Even I was one of them. So, I thought why not we paint in air ? Or is it possible?
yes, it is possible, I have created a paint app where you can paint in air. 

- [x] LGM-SOC'21 Participant 
- [ ] Contributor

Closes: #316

## Type of change

What sort of change have you made:
<!--
Example how to mark a checkbox:-
- [x] My code follows the code style of this project.
-->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?
I have tested in my loacl computer.

## Checklist:
<!--
Example how to mark a checkbox:-
- [x] My code follows the code style of this project.
-->
- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly whereever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots

 Original           | Updated
 :--------------------: |:--------------------:
![Uploading screen-capture (7).gif…]()
the file seems too big. kindly please refer the link : https://github.com/manognyaa/Awesome_Python_Scripts/tree/main/ImageProcessingScripts/paintme/Images| Updated Screenshot |

prathimacode-hub reviewed (changes requested):
Small changes. Kindly rename the project folder to "Paint Me". And project file as "paint_me.py". Add libraries included in "requirements.txt" file.

@manognyaa

manognyaa commented:
> Small changes. Kindly rename the project folder to "Paint Me". And project file as "paint_me.py". Add libraries included in "requirements.txt" file.
> 
> @manognyaa

Hello mam,
I have updated the folder name also added the requirement.txt file
please guide me, if any changes required  @prathimacode-hub mam

prathimacode-hub reviewed (approved):
It's set perfectly. No changes needed. Yayy, good to see your learning journey. Your efforts are highly appreciated. 🤩👌🙌@manognyaa
